client/src/components/Overview.jsx

```diff
@@ -3,26 +3,40 @@ import React from 'react';
 import ProductDetails from './ProductDetails.jsx'
 import ImageGallery from './ImageGallery.jsx'
 
-const Overview = (props) => {
-  // TEMPORARY STYLING
-  var styles = {
-    width: "maxWidth",
-    display: "flex",
-    flexDirection: "row",
-    justifyContent: "center"
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+  root: {
+    border: "dotted 1px grey",
+  },
+
+  prodOverview: {
+    padding: "20px",
   }
+});
+
+const Overview = (props) => {
+  const classes = useStyles();
 
   return (
-    <div>
-      <div style={styles}>
-        <ImageGallery />
-        <ProductDetails />
-      </div>
-      <div>
-        <h2>Product Overview/Description</h2>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. A pellentesque sit amet porttitor eget. Commodo elit at imperdiet dui accumsan. Et leo duis ut diam quam nulla. Vitae ultricies leo integer malesuada. Diam volutpat commodo sed egestas egestas fringilla phasellus. Consectetur adipiscing elit duis tristique sollicitudin nibh sit amet. Nunc mattis enim ut tellus elementum sagittis vitae et. Vel turpis nunc eget lorem dolor sed viverra. Et malesuada fames ac turpis egestas maecenas pharetra convallis. Lorem sed risus ultricies tristique nulla aliquet enim tortor at. Ultricies mi quis hendrerit dolor magna eget est lorem.</p>
-      </div>
-    </div>
+      <Grid className={classes.root} container direction="row" justifyContent="flex-start" alignItems="flex-start" >
```

Comment:
If you look at the Grid API, you can see that direction defaults to "row" and justifyContent defaults to "flex-start" so those are unnecessary to add